tests/run/i7031.scala

```diff
@@ -2,8 +2,7 @@
   val a = 5
   val x = 1
 
-    + //
-    `a` * 6
```

Comment:
Hey,

Are you sure that it is related to this issue?

I feel that it introduces another one similar to this: https://github.com/lampepfl/dotty/pull/7031

Reply:
This issue is interesting. It worked somewhat by accident before. First, if the backticked "`a`" gets changed to "a" this never compiles. We do not skip a newline after a prefix operator, so we have a prefix operator `+` with nothing that follows it --> error. What happened previously was that the "`a`" was interpreted as a leading infix operator, so the lexical analyzer saw a noun `+`, followed by an operator "`a`", and did not insert a new line. Then the parser made different sense of it and reclassified everything. But now we do not interpret "`a`" as a leading operator anymore, so we get an error (as we should).

Reply:
I see.

I've also plaid a bit this with scala 2 and can't compile it.

Reply:
Aha. I stared at that test while bringing `-Xsource:3` into better alignment. Thanks!